src/main/java/com/example/demo/repository/UserRepository.java

```diff
@@ -0,0 +1,9 @@
+package com.example.demo.repository;
+
+import com.example.demo.model.User;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface UserRepository extends JpaRepository<User, Long> {
+    Optional<User> findByEmail(String email);
```

Comment:
Do you need to customize this method, it should be implemented inside JPA, just use it

Reply:
I didn`t find that  method inside JPA